webknossos/webknossos/client/api_client/datastore_api_client.py

```diff
@@ -72,10 +73,13 @@ def dataset_reserve_manual_upload(
         self,
         dataset_announce: ApiDatasetAnnounceUpload,
         token: str | None,
-    ) -> None:
+        require_unique_name: bool = False,
+    ) -> ApiDatasetManualUploadSuccess:
         route = "/datasets/reserveManualUpload"
-        query: Query = {"token": token}
-        self._post_json(route, dataset_announce, query)
+        query: Query = {"token": token, "requireUniqueName": require_unique_name}
```

Comment:
Could you please make `require_unique_name` part of `ApiDatasetAnnounceUpload`? Otherwise the backend needs to extract the info not only from the post body but also from the query string. I would prefer to have all the info in the body (excluding the token which is needed as a query parameter) :pray: